acra-censor/acra-censor_test.go

```diff
@@ -860,9 +864,10 @@ func testBlacklistValuePattern(t *testing.T) {
 	}
 
 	blockableQueries = []string{
-		"SELECT a, b FROM t WHERE ID = 'someValue_testValue_1234567890'",
-		"SELECT a, b FROM t WHERE ID = 'someValue'",
+		"SELECT a, b FROM t WHERE ID = 1",
+		"SELECT a, b FROM t WHERE ID = 1.0",
```

Comment:
more important to check cases with >1 blockable values %%value%%
can we check queries like `where a= %%value%% and b=%%value%% or c is Null` ?